spec.md

```diff
@@ -164,49 +202,100 @@ echo "SPDX-License-Identifier: MIT" > file.txt
 # REUSE-IgnoreEnd
 ```
 
-### In-line snippet comments
-
-If a copyright and/or licensing info is to apply only to a certain snippet
-instead of the whole file, SPDX snippet tags SHOULD be used (as defined in [SPDX
-Specification, Annex H](https://spdx.github.io/spdx-spec/v2.3/file-tags/#h3-snippet-tags-format)).
-
-Such an annotated snippet block MUST start with `SPDX-SnippetBegin` to mark its
-beginning and end with `SPDX-SnippetEnd` to mark the snippet's end.
-
-Do note that SPDX snippet tags MUST start with `SPDX-Snippet`, meaning that the
-correct copyright notice in a snippet is `SPDX-SnippetCopyrightText`.
-
-Example:
-
-```
-# SPDX-SnippetBegin
-# SPDX-License-Identifier: MIT
-# SPDX-SnippetCopyrightText: 2022 Jane Doe <jane@example.com>
-
-{$snippet_code_goes_here}
-
-# SPDX-SnippetEnd
+### REUSE.toml
+
+Copyright and Licensing Information MAY be associated with a file through a
+`REUSE.toml` file. The intended use case of this method is large directories where
+including a comment header in each file (or in `.license` companion files) is
+impossible or undesirable.
+
+A `REUSE.toml` file MAY be located in any directory, and can cover files that are
+within its directory or deeper. You MAY have multiple `REUSE.toml` files in
+different directories.
+
+The `version` key (REQUIRED) MUST have an integer value representing the schema
+version of the file. This specification describes version 1 of `REUSE.toml`.
+
+Each `[[annotations]]` table represents an association of Copyright and
+Licensing Information to zero or more Covered Files. It has the following keys:
+
+- `path` (REQUIRED), a string or list of strings representing paths. A path MUST
+  use forward slashes as path separators. A path SHOULD resolve to one or more
+  Covered Files relative to the `REUSE.toml` file's directory. A path MAY use
+  globbing to match several Covered Files in a single expression. These are the
+  globbing and matching rules:
+  - `*` matches everything except forward slashes (i.e. path separators).
+  - `**` and `**/` match everything including forward slashes (i.e. path
+    separators).
+  - In order to escape an asterisk and include it verbatim, prefix it with `\\`.
+    You cannot prefix it with `\` because that is invalid TOML. In order to
+    include a backslash verbatim, use `\\\\`. `\\` followed by any other
+    character is functionally equal to just typing that character.
```

Comment:
It may be worth to mention that using `..` for parent directories is not allowed? So only paths in the file's root or child directories are allowed.

Reply:
This is made explicit in an earlier paragraph:

> A REUSE.toml file MAY be located in any directory, and can cover files that are within its directory or deeper

should it be more explicit?

Reply:
Ah, I somehow missed that. Perhaps adding a sentence behind: "It MUST NOT cover files in parent directories."?

Reply:
I touched this up in 7bddcad2dc7e9a2cb19c63a2d739da397f50f880. It was tricky to get the language right, but I think it's all right.